pkg/kwokctl/snapshot/save.go

```diff
@@ -37,7 +37,7 @@ import (
 )
 
 // Save saves the snapshot of cluster
-func Save(ctx context.Context, kubeconfigPath string, w io.Writer, resources []string, impersonateConfig rest.ImpersonationConfig) error {
```

Comment:
Do we define a structure to put all these extra parameters in?

Reply:
Do you mean the pager configuration or all the parameters?

Reply:
I mean the `rest.ImpersonationConfig` and the pager configuration